.circleci/config.yml

```diff
@@ -1,37 +1,319 @@
-# Javascript Node CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
+# Configuration file for https://circleci.com/gh/angular/flex-layout
```

Comment:
Maybe also note that this is based on Material's configuration?

Reply:
Yes definitely. I'll also credit you explicitly because this work is fantastic.